packages/range/README.md

```
@@ -10,8 +10,9 @@
 
 Control Flow Primitives for displaying a number range or given number of elements.
 
+- [`createNumericRange`](#createnumericrange) - Reactively generates a number array for a given range. Convenient companion to the built-in `<For>`.
 - [`repeat`](#repeat) - Primitive for mapping a number of elements. Underlying helper for the [`<Repeat>`](#repeat-1) control flow.
-- [`<Repeat>`](#repeat-1) - Control Flow Component for displaying a number of elements.
+- [`<Repeat>`](#repeat-1) - Control Flow Component for displaying a number of elements. ⚠️ Deprecated — use Solid 2.0's built-in `<Repeat>`.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)